Ports/zsh/zshrc

```diff
@@ -0,0 +1,46 @@
+###############
+# Completions #
+###############
+
+# Completions
+autoload -U compinit
+compinit -C
+
+# Arrow key menu for completions
+zstyle ':completion:*' menu select
+
+# Case-insensitive (all),partial-word and then substring completion
+zstyle ':completion:*' matcher-list 'm:{a-zA-Z}={A-Za-z}' 'r:|[._-]=* r:|=*' 'l:|=* r:|=*'
+
+###########
+# Aliases #
+###########
+
+# Set up aliases
+alias ll='ls -l'
+
+# Autocomplete command line switches for aliases
+setopt completealiases
+
+###########
+# History #
+###########
+
+# number of lines kept in history
+export HISTSIZE=1000
+# number of lines saved in the history after logout
+export SAVEHIST=1000
+# location of history
+export HISTFILE=~/.zsh_history
```

Comment:
These should definitely **not** be exported: there is no reason to export them, and exporting them would just possibly break the history of `bash` or other shells (and likely also overwrite, corrupt, or wipe `~/.zsh_history`) if you run them from `zsh` and if they don't overwrite those variables in their bashrc file or equivalent.

Reply:
Good catch!